pkgs/development/python-modules/manim-slides/default.nix

```diff
@@ -0,0 +1,93 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+
+, manim
+, ffmpeg
+
+, av
+, click
+, click-default-group
+, jinja2
+, lxml
+, numpy
+, opencv4
+, pillow
+, pydantic
+, pydantic-extra-types
+, python-pptx
+, qtpy
+, requests
+, rich
+, rtoml
+, tqdm
+, pyqt6
+
+  # Optional dependencies
+, ipython
+
+  # Hooks
+, pdm-backend
+, pythonRelaxDepsHook
+
+  # As Module or application?
+, withGui ? false
+}:
+buildPythonPackage rec {
+  pname = "manim-slides";
+  format = "pyproject";
+  version = "5.1.3";
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "jeertmans";
+    repo = "manim-slides";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-WZR95swapT2Fbu6mbuHLjMu3Okq/wKFabzN7xpZw0/g=";
+  };
+
+  nativeBuildInputs = [ pdm-backend pythonRelaxDepsHook ];
+
+  pythonRemoveDeps = [ "opencv-python" ];
+
+  pythonRelaxDeps = [ "rtoml" "qtpy" ];
+
+  propagatedBuildInputs = [
+    av
+    click
+    click-default-group
+    jinja2
+    lxml
+    numpy
+    opencv4
+    pillow
+    pydantic
+    pydantic-extra-types
+    python-pptx
+    qtpy
+    requests
+    rich
+    rtoml
+    tqdm
+
+    # dependency of qtpy (could also be pyqt5)
+    pyqt6
+    # avconv is a potential alternative
+    ffmpeg
+    # This could also be manimgl, but that is not (yet) packaged
+    manim
+  ] ++ lib.lists.optional withGui ipython;
```

Comment:
`!withGui` according to the pytoml

I asked about the other deps here: https://github.com/jeertmans/manim-slides/issues/393

Could just comment the issue and move on

Reply:
Oh OK- I just don't know how pytoml works. I think pyqt6 should be on the other branch

Reply:
I *think* I have the dependencies sorted out correctly now. But I'm not so sure about `qtpy`, from the pytoml file, it's just a normal dependency for every use case?